indexer/packages/postgres/src/lib/parent-subaccount-helpers.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  CHILD_SUBACCOUNT_MULTIPLIER,
+  MAX_PARENT_SUBACCOUNTS,
+} from '../constants';
+
+export function getParentSubaccountNum(childSubaccountNum: number): number {
```

Comment:
Any reason to move this to a separate file?

Reply:
Is this so that it can be used by both comlink and socks?

Reply:
Yes, that's the reason.